proposals/views/talks.py

```diff
@@ -31,13 +32,59 @@ def get_queryset(self):
         return super().get_queryset().filter(
             state=Proposal.PROPOSAL_ACCEPTED).order_by('?')
 
+
+    def get_tags(self, talks):
+        tags = talks.filter(tags__isnull=False).values_list('tags', flat=True).distinct().order_by()
+        tag_list = []
+        [tag_list.extend(tag) for tag in tags]
```

Comment:
O tai tags nėra tas pats kas tag_list? Na bet jei nėra tai galima rašyti tag_list = [tag for tag in tags]